src/app/GitExtensions.Extensibility/Git/IGitModule.cs

```diff
@@ -21,7 +21,8 @@ public interface IGitModule
 
     IReadOnlyList<IGitRef> GetRefs(RefsFilter getRef);
     IEnumerable<string> GetSettings(string setting);
-    IEnumerable<INamedGitItem> GetTree(ObjectId? commitId, bool full);
+    IEnumerable<IObjectGitItem> GetTree(ObjectId? commitId, bool full, CancellationToken cancellationToken = default);
+    IEnumerable<IObjectGitItem> GetGitItemTree(ObjectId? commitId, bool full, string? fileName, CancellationToken cancellationToken = default);
```

Comment:
```suggestion
    IEnumerable<IObjectGitItem> GetTree(ObjectId? commitId, bool full, string fileName = "", CancellationToken cancellationToken = default);
```
Here Igor's question "Which method to call?" applies for sure.